packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-custom-pwd-recovery-mfa/main/java/beforeyoubegin.md

```diff
@@ -0,0 +1,7 @@
+1. Create an Okta org and configure it as described in [Set up your Okta org for a password factor only use case](/docs/guides/oie-embedded-common-org-setup/java/main/#set-up-your-okta-org-for-a-password-factor-only-use-case).
+1. [Set up the SDK and Sign-In Widget](/docs/guides/oie-embedded-common-download-setup-app/java/main/)
+1. Build an app with the Embedded Sign-In Widget or SDK.
+    * For embedded Sign-In Widget based apps, implement the [basic sign-in flow](/docs/guides/oie-embedded-widget-use-case-basic-sign-in/java/main/).
```

Comment:
But "embedded apps" isnt a thing per se, only apps using the embedded model, and the embedded model means using either an embedded Sign-In Widget or an "embedded SDK" (which is a very strange way of putting it because you don't embed SDKs you just build with them, but the terms are laid out inthe overview for the Sign-In Section fo teh guides - https://developer.okta.com/docs/guides/sign-in-overview/main/#choose-your-auth

Happy to change to "For apps using an embedded Sign-In Widget" and "For apps using the SDK"?